internal/server/property_value.go

```diff
@@ -107,36 +106,7 @@ func getPropertyValuesHelper(
 ) (map[string][]*Node, error) {
 	rowList := buildPropertyValuesKey(dcids, prop, arcOut)
 	// Add base cache data
-	nodeMap, err := readPropertyValues(ctx, store, rowList)
-	if err != nil {
-		return nil, err
-	}
-	// Add branch cache data
-	branchNodeMap, err := readPropertyValues(ctx, store, rowList)
-	if err != nil {
-		return nil, err
-	}
-
-	for dcid := range branchNodeMap {
-		branchNodes := branchNodeMap[dcid]
-		baseNodes, exist := nodeMap[dcid]
-		if !exist {
-			nodeMap[dcid] = branchNodes
-		} else if len(branchNodes) > 0 {
-			// Merge branch cache into base cache.
-			itemKeys := mapset.NewSet()
-			for _, n := range baseNodes {
-				itemKeys.Add(n.Dcid + n.Value)
-			}
-			for _, n := range branchNodes {
-				if itemKeys.Contains(n.Dcid + n.Value) {
-					continue
-				}
-				nodeMap[dcid] = append(nodeMap[dcid], n)
-			}
-		}
-	}
-	return nodeMap, nil
+	return readPropertyValues(ctx, store, rowList)
```

Comment:
Add comments for why only base cache?

Reply:
Done. This is based on past inspection of the cache. Need to do this for current data though. But since current branch cache is targeted on new stats (without addition of schema etc), so this assumption should still holds.